src/lib/renderer/Markdoc.svelte

```diff
@@ -0,0 +1,15 @@
+<script>
+  export let doc = ""
+  export let config = {}
+  export let components = new Map
+
+  import Markdoc from "@markdoc/markdoc"
+  import Tags from "./Tags.svelte"
+  import { addFrontmatter } from "../utils.js"
+
+  const ast = Markdoc.parse(doc)
+  const configWithFrontmatter = addFrontmatter(ast, config)
+  const content = Markdoc.transform(ast, configWithFrontmatter)
```

Comment:
Previous three lines should probably be in a reactive block `$: { .... }`. since user can change props `doc`, `config`, etc..

Reply:
100% yep